docker/obdemo-rs/Dockerfile

```diff
@@ -1,12 +1,17 @@
-FROM openjdk:8-jre-alpine
+FROM google/cloud-sdk:alpine
```

Comment:
Change the image `openjdk:8-jre-alpine` to `google/cloud-sdk:alpine` to use `gsutil` instead of `ADD` to avoid get corrupted files from `google bucket`, more fast and secure.